NewForce-Cohort-4/react-nutshell-spidereye
# Jp events

## What is it?
This pull request contains an Events component that allows the user to add, edit, and delete an event and see all of their events in the DOM. 

## How to test it
Register for an account and login as a user. Go to the "Events" link in the navbar. Click the "Add new event" button and a form will be displayed. Input the event name, location, and date then click the "Save" button. The event will be saved to the database and will be displayed in the DOM. The events are sorted by date with the next event printed in bold with a larger border to indicate that this event is next to occur. Add several events to see how they are sorted. Each event card also has an edit button that allows the user to edit the event, and a delete button that allows the user to delete that event from the database.
